index.html

```diff
@@ -0,0 +1,227 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Products</title>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/product.css">
+    <link rel="stylesheet" href="css/footer.css">
+    <link rel="manifest" href="site.webmanifest">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+  </head>
+  <body>
+  <header>
+    <div class="nav-h2">
+      <svg xmlns="http://www.w3.org/2000/svg" width="72" height="32" viewBox="0 0 72 32">
+        <g id="Group_17915" data-name="Group 17915" transform="translate(-48 -16)">
+          <g id="Component_839_7" data-name="Component 839 – 7" transform="translate(48 16)">
+            <path id="Path_12189" data-name="Path 12189" d="M-13238.507-18478.545l-9.994,32h6.43l9.973-32Z" transform="translate(13258.223 18478.545)" fill="#040138"/>
+            <path id="Subtraction_28" data-name="Subtraction 28" d="M14.248,31.95,4.476,0H6.033l9.634,31.515-1.418.434ZM12,31.95,2.229,0H3.784l9.634,31.515L12,31.95Zm-2.23,0L0,0H1.555l9.634,31.515-1.421.434Z" transform="translate(0 0.049)" fill="#040138"/>
+          </g>
+          <rect id="Rectangle_21603" data-name="Rectangle 21603" width="40" height="18" rx="4" transform="translate(80 16)" fill="#040138"/>
+          <text id="DisplayLarge" transform="translate(85 29)" fill="#fff" font-size="12" font-family="Roboto-Black, Roboto" font-weight="800"><tspan x="0" y="0">BETA</tspan></text>
+        </g>
+      </svg>
+    <nav >
+      <ul>
+        <li class="nav-h2-h1-selected"><a href="/">Inventory<i class="icon-down"></i></a></li>
```

Comment:
Always try and breakdown your code for easy visibility

```suggestion
        <li class="nav-h2-h1-selected">
            <a href="/">
                Inventory
                <i class="icon-down"></i>
            </a>
        </li>
```